ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -86,6 +86,13 @@ open class NavigationBarTitleAccessory: NSObject {
     }
 }
 
+// MARK: - NavigationBarBackButtonDelegate
+/// Handles presses from a `NavigationBar`'s makeshift back button.
```

Comment:
nit: maybe "custom" instead of "makeshift"

Reply:
Reworded.